src/main/java/productDependency/CreateDependentCategoryPage.java

```diff
@@ -0,0 +1,974 @@
+		log.info("Verifying the login button is available or not");
+		WaitClass.WaitForElementisDisplay(driver, 5, clickLoginButton);
+		Assert.assertTrue(clickLoginButton.isDisplayed());
+		clickLoginButton.click();
+	}
+	
+	@FindBy(how = How.XPATH, using = "//*[@id='menu.link.products']/a")
+	private WebElement clickProductsTab;
+
+	/**
+	 * Method to click on Products tab after successful login.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickProductsTab() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		log.info("Click on Products Tab after successful login");
+		JavaScriptExec.sleep();
+		Assert.assertTrue(clickProductsTab.isDisplayed());
+		clickProductsTab.click();
+
+	}
+
+	@FindBy(how = How.XPATH, using = "//a[@class='submit add']//*[text()='Add Category']")
+	private WebElement clickAddCategoryButton;
+
+	/**
+	 * Method to click on Add Category Button.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickAddCategoryButton() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		log.info("Click on Add Category Button");
+		navigateBottom();
+		Assert.assertTrue(clickAddCategoryButton.isDisplayed());
+		clickAddCategoryButton.click();
+
+	}
+
+	@FindBy(how = How.XPATH, using = "//input[@name='description']")
+	private WebElement enterCategoryName;
+
+	/**
+	 * Method to Enter Category Name.
+	 * 
+	 * @throws IOException
+	 */
+	public void enterCategoryName() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		log.info("Enter Category Name");
+		Assert.assertTrue(enterCategoryName.isDisplayed());
+		enterCategoryName.sendKeys(BasePage.getCellData(xlsxName, sheetName, 3, 3));
+
+	}
+	
+	
+	@FindBy(how = How.XPATH, using = "//a[@class='submit save']//*[text()='Save Changes']")
+	private WebElement clickSaveChangesButton;
+
+	/**
+	 * Method to click on Save Changes Button.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickSaveChangesButton() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		log.info("Click on Save Changes Button");
+		Assert.assertTrue(clickSaveChangesButton.isDisplayed());
+		clickSaveChangesButton.click();
+
+	}
+
+	@FindBy(how = How.XPATH, using = "//div[@class='msg-box successfully']//*[text()='Done']")
+	private WebElement verifyConfirmationMsg;
+
+	/**
+	 * Method to verify Product is created Successfully.
+	 * 
+	 * @throws IOException
+	 */
+	public void verifyConfirmationMsg() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		log.info("Verifying if Product is created Successfully or not");
+		JavaScriptExec.sleep();
+		Assert.assertTrue(verifyConfirmationMsg.isDisplayed(),
+				"Assert Failed as its unable to search text in Logged in Page");
+	}
+	
+	private WebElement selectCategoryName;
+
```

Comment:
@mayankchhajer When you are not defining any WebElement with the XPath, it means they are unused so remove them.

Reply:
Done